worker/internal/benthos/config.go

```diff
@@ -58,7 +58,12 @@ type PipelineConfig struct {
 }
 
 type ProcessorConfig struct {
-	Mutation string `json:"mutation" yaml:"mutation"`
+	Mutation   string           `json:"mutation" yaml:"mutation"`
+	Javascript JavascriptConfig `json:"javascriptConfig" yaml:"javascriptConfig"`
```

Comment:
Now that we have two different types of processors, this should be updated to omitempty for both of them.

That way if someone provides only mutations, or only javascript configs, the other does not appear in the yaml.

```suggestion
	Mutation   *string           `json:"mutation,omitempty" yaml:"mutation,omitempty"`
	Javascript *JavascriptConfig `json:"javascriptConfig,omitempty" yaml:"javascriptConfig,omitempty"`
```

Also, I don't think that json is correct. According to benthos the serialized prop should j ust be `javascript`.

<img width="992" alt="image" src="https://github.com/nucleuscloud/neosync/assets/2420177/225b1901-812e-4eca-a0a2-fe328ad70bd6">

Which makes this:

```suggestion
	Mutation   *string           `json:"mutation,omitempty" yaml:"mutation,omitempty"`
	Javascript *JavascriptConfig `json:"javascript,omitempty" yaml:"javascript,omitempty"`
```

Reply:
ah yeah good catch, it's just javascript